### PR TITLE
Add a `BadgedLink` action to `EmptyState`

### PR DESCRIPTION
**Pull request by BeirlaenAaron**
### Description

Adds an action prop to `EmptyState` that renders a `BadgedLink`

image

### CODE REVIEW

**Review by jelledc — DISMISSED** [review]

File: src/components/emptyState/EmptyState.tsx [hunk 1]
```diff
@@ -11,12 +11,15 @@ import theme from './theme.css';
...
 import { SIZES } from '../../constants';
+import BadgedLink, { BadgedLinkProps } from '../badgedLink';
+import { IconAddSmallOutline } from '@teamleader/ui-icons';
 
 export interface EmptyStateProps extends Omit<BoxProps, 'size'> {
   hidePointer?: boolean;
   metaText?: ReactNode | string;
   size?: Exclude<typeof SIZES[number], 'tiny' | 'fullscreen' | 'smallest' | 'hero'>;
   title?: ReactNode | string;
+  action?: Omit<BadgedLinkProps, 'icon' | 'inheric'> & { children: ReactNode };
```
Comment (qubis741) [inline src/components/emptyState/EmptyState.tsx:22]: you have typo - inheric instead of inherit

Comment (BeirlaenAaron) [reply src/components/emptyState/EmptyState.tsx]: copied this over from somewhere else, didn't notice it :sweat_smile:

**Comment by JorenSaeyTL** [comment]
MCing ⌛ ...

**Review by eniskraasniqi1 — APPROVED** [review]

**Comment by JorenSaeyTL** [comment]
MCed ✅

**Review by stefaandevylder — APPROVED** [review]

**Review by lowiebenoot — APPROVED** [review]

**Review by qubis741 — APPROVED** [review]

**Review by KristofColpaert — APPROVED** [review]

**Pull request merged**